modules/dnn/test/test_model.cpp

```diff
@@ -331,7 +331,7 @@ TEST_P(Test_Model, DetectRegion)
 
     double confThreshold = 0.24;
     double nmsThreshold = (target == DNN_TARGET_MYRIAD) ? 0.397 : 0.4;
-    double scoreDiff = 8e-5, iouDiff = 1e-5;
+    double scoreDiff = 1e-4, iouDiff = 1e-5;
```

Comment:
The arm instruction `vfmaq_laneq_f32` is only supported at ARMv8. So I use the `vmlaq_n_f32` instead. I adjusted the thresholds here and below because I found the `vmlaq_n_f32` will generate very little different results than `vmlaq_n_f32`.

Reply:
On my M1 chip, the `vmlaq_n_f32 ` will be parsed as follows:
```
+0x198	fmul.4s             v24, v22, v20[0]
+0x19c	fadd.4s             v3, v3, v24
```
And the `vfmaq_laneq_f32 ` will be parsed as follows:
```
fmla.4s             v16, v2, v14[3]
```